tests/components/pyload/conftest.py

```diff
@@ -72,7 +72,7 @@ def pyload_config() -> ConfigType:
 
 
 @pytest.fixture
-def mock_pyloadapi() -> Generator[AsyncMock, None, None]:
+def mock_pyloadapi() -> Generator[AsyncMock]:
```

Comment:
```suggestion
def mock_pyloadapi() -> Generator[Mock]:
```

For this one, it would be really good to use `Generator[MockOf[PyLoadAPI]]` but it's not available yet.